packages/client-account/test/Account.test.ts

```diff
@@ -31,6 +31,7 @@ describe("AccountClientImpl", () => {
     const result = await pipe(
       program,
       Effect.provide(Account.defaultLayer),
+      Effect.scoped,
```

Comment:
this becomes unnecessary if you use `Layer.scoped` when constructing an instance layer